packages/amplify-category-geo/src/service-walkthroughs/deviceLocationTrackingWalkthrough.ts

```diff
@@ -111,3 +114,126 @@ export const deviceLocationTrackerAccessWalkthrough = async (
   updatedParameters.roleAndGroupPermissionsMap = selectedGroupPermissions;
   return updatedParameters;
 };
+
+/**
+ * deviceLocationTrackerAdvancedWalkthrough
+ */
+export const deviceLocationTrackerAdvancedWalkthrough = async (
+  context: $TSContext,
+  parameters: Partial<DeviceLocationTrackingParameters>,
+): Promise<Partial<DeviceLocationTrackingParameters>> => {
+  let updatedParameters = { ...parameters };
+  updatedParameters.positionFiltering = deviceLocationTrackingPositionFilteringTypes['Time-based'];
+  const selectedAdvancedSetting = await prompter.pick<'one', string>(
+    `Here are the default advanced settings. Select a setting to edit or continue (Use arrow keys)`,
+    Object.values(deviceLocationTrackingAdvancedSettings),
+    { returnSize: 'one' },
+  );
+  switch (selectedAdvancedSetting) {
+    // Grant users access to devices other than their own
+    case deviceLocationTrackingAdvancedSettings.grantOtherAccess:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerOtherAccessWalkthrough(updatedParameters));
+      break;
+    // Link Geofence Collection Question
+    case deviceLocationTrackingAdvancedSettings.linkGeofenceCollection:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerGeofenceLinkingWalkthrough(updatedParameters));
+      break;
+    // KMS Settings
+    case deviceLocationTrackingAdvancedSettings.addKMSSettings:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerKMSSettingsWalkthrough(context, updatedParameters));
+      break;
+    // Position filtering method settings
+    case deviceLocationTrackingAdvancedSettings.setPositionFilteringMethod:
+      updatedParameters = merge(updatedParameters, await deviceLocationTrackerFilteringMethodWalkthrough(updatedParameters));
+      break;
+    default:
+      printer.error('something went wrong...');
+  }
+  return updatedParameters;
+};
+
+const deviceLocationTrackerOtherAccessWalkthrough = async (
+  parameters: Partial<DeviceLocationTrackingParameters>,
+): Promise<Partial<DeviceLocationTrackingParameters>> => {
+  const updatedParameters = { ...parameters };
+  printer.info('Users in this group can only access their own device by default. Learn more at ...');
```

Comment:
Checking with PM for the doc link

Reply:
Todo?

Reply:
Yes, this is the only thing pending for now.
Waiting on Location services team for the doc url.

Reply:
Added doc url